src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -837,7 +837,7 @@ private string ManifestComment(string insert, StreamWriter streamWriter)
         /// </summary>
         protected override void EndProcessing()
         {
-            // Win8: 264471 - Error message for New-ModuleManifest �ProcessorArchitecture is obsolete.
+            // Win8: 264471 - Error message for New-ModuleManifest - ProcessorArchitecture is obsolete.
```

Comment:
ProcessorArchitecture is a parameter, so it should be written as:

// Win8: 264471 - Error message for New-ModuleManifest -ProcessorArchitecture is obsolete.